README.md

```diff
@@ -124,6 +124,13 @@ skip_verify=false
 url=""
 secret=""
 
+[stash]
+url = ""
+api = ""
+secret = ""
+private-key = ""
+hook = "de.aeffle.stash.plugin.stash-http-get-post-receive-hook%3Ahttp-get-post-receive-hook"
```

Comment:
@vassilevsky These are the configuration options for the Stash plugin.
See `Stash` section further along in the README.md for more information